docker-assets/check-dependent-services.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/bin/true
```

Comment:
o_O  just a placeholder?

Reply:
Yeah, this is called by the entrypoint in the podified images so that we wait for PG / memcached to start up before trying to access them.

We externalized it so that different images could have different dependencies, but this image doesn't do anything like that, so if we didn't overwrite it here I think we would get errors in the logs from [the base image implementation](https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app-frontend/docker-assets/check-dependent-services.sh) which would end up executing `ncat` with invalid args.